README.md

```diff
@@ -1 +1,145 @@
-# fe-hometask
\ No newline at end of file
+# React & Rockets
+
+**Description:** Challenge for Adjust Frontend Developer
+
+**Owner:** [adjust](https://github.com/adjust)
+
+**Contacts:** frontend-hiring@adjust.com
+
+---
+
+## Welcome to our little coding exercise! 👋
+
+Here you will be given the opportunity to play with JavaScript and Rockets in the same project. For this, we recommend you to block 2-3 hours of your time to complete it.
+
+**Please make sure you have read this page entirely, before starting the challenge**. If you have any doubts, please reach out to us by [opening an issue](#how-to-request-help) as described at the bottom of this page.
+
+**Importantly**, we would like to give all candidates taking this test the same opportunity to solve the exercise in their own way. Because of this, **please do not fork or share this repo (nor your solution) with anybody 🙏🏻**
+
+<img align="center" src="https://i.imgur.com/ekyJNd9.jpg" width="600">
+
+## Instructions
+
+1. Push your solution to a **private repo** in your **personal Github account**
+2. When you are ready for us to take a look, please add [adjust-frontend-hiring][adjust-frontend-hiring] (GitHub user) as collaborator
+
+## General notes
+
+- The final solution must be a working React application
+- You can use any library you see fit
+- Bringing in best practices such as tests, types, linting will be appreciated
+
```

Comment:
Some candidates leave a lot of unnecessary comments. We could add the following:

```suggestion
- Bringing in best practices such as tests, types, linting will be appreciated
- It's not necessary to heavily comment the code, please only leave comments where you think they are necessary to understand the related code. 
```

Reply:
> Some candidates leave a lot of unnecessary comments. We could add the following:

Tbh I don't clearly understand why this is a problem for the home task

Reply:
It's not a problem per se, although the more "expectations" we set, the easier it will be for the candidates to align with our expectations.
For an abstract home task, they don't know if they should add notes on every step or not. If we explicitly mention that they don't need to spend time on this, they won't have doubts when submitting a solution. Otherwise they might think "Ok I better comment every line of code to show my understanding of it". We don't need this and can save them a bit of time.